README.md

```diff
@@ -38,6 +41,14 @@ Once the migration is complete, check configuration by starting the server. Star
 
 ![screenshot_home.jpg](./images/screenshot_home.png)
 
+## Running Docker locally
+
+```
+cp deployment/Dockerfile .
+docker build . -t umf
```

Comment:
No need for copying the Docker file. The `cp` command can be eliminated and the docker command can be changed to:
```
docker build -f deployment/Dockerfile -t umf .
```
This also means that there's no need to add `Dockerfile` to `.gitignore` since no copying should be happening.